src/main/java/de/unijena/cheminf/fragment/fingerprint/performanceTest/PerformanceTest.java

```diff
@@ -0,0 +1,419 @@
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package de.unijena.cheminf.fragment.fingerprint.performanceTest;
+
+import de.unijena.cheminf.fragment.fingerprint.FragmentFingerprinter;
+import org.openscience.cdk.fingerprint.IBitFingerprint;
+import org.openscience.cdk.fingerprint.ICountFingerprint;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * A class for testing the performance of the fragment fingerprinter.
+ *
+ * @author Betuel Sevindik
+ */
+public class PerformanceTest {
+    //<editor-fold defaultstate="collapsed" desc="Private static final constants">
+    /**
+     * Name of file for logging occurred exceptions
+     */
+    private static final String EXCEPTIONS_LOG_FILE_NAME = "Exceptions_Log.txt";
+    /**
+     * Name of file for writing results
+     */
+    private static final String RESULTS_FILE_NAME = "Results.txt";
+    /**
+     * Name of CSV file with the results of the performance test of the bit fingerprints.
+     */
+    private static final String CSV_BIT_FINGERPRINT_PROCESS_RESULT_FILE_NAME = "CSV_BIT_PROCESS_TIME";
+    /**
+     * Name of CSV file with the results of the performance test of the count fingerprints.
+     */
+    private static final String CSV_COUNT_FINGERPRINT_PROCESS_RESULT_FILE_NAME = "CSV_COUNT_PROCESS_TIME";
+    /**
+     * Name of the CSV file with the results of the generated bit arrays.
+     */
+    private static final String BIT_ARRAY_RESULT_FILE_NAME = "BIT_ARRAY_TIME";
```

Comment:
Not every informative. These are basically the bit fingerprints, right? Please name it accordingly. 
And you don't safe the generated count fingerprints in a file? Could that easily be added?